content/en/news/2016-12-23-new-domain.md

```diff
@@ -0,0 +1,28 @@
+---
+title: New Domain Name
+date: 2016-12-23
+author: Sydney
+translator:
+summary: Info on our new Domain name
+---
+Hello all, hope everyone is having a happy holiday.
+
+On behalf of the Secretchronicles team, we are quite happy to announce that we have purchased
+the secretchronicles.org domain name, and for the most part have it fully implemented. We will continue to
+have the secretchronicles.de domain for a while yet, but thanks to xet7's hard work, it now redirects to our
+new domain.
+
+We are also quite proud to have enabled HTTPS on our site and all of its resources in this process.
+
+If you use email to post to our forum or mailinglist, please update your address book from
+`secretchronicles.de` to
+`secretchronicles.org`.
+
+Also, if you run into any issues feel free to contact us on [our IRC channel][1] or email us.
+
+Have a great holiday!
+
+-Sydney(sydney_untangle)
+On behalf of the TSC team
+
+[1]: irc://irc.freenode.net/secretchronicles
```

Comment:
Are you sure the `irc://` link scheme does not require the `#` identifier before the channel? I honestly don't know, but there are channels that start with multiple `##`, so one couldn't link to them otherwise. You might want to find this out.